src/main/java/org/example/service/BuildingService.java

```diff
@@ -0,0 +1,49 @@
+package org.example.service;
+
+import org.example.dto.BuildingDTO;
+import org.example.entity.Apartment;
+import org.example.entity.Building;
+import org.example.repository.ApartmentRepository;
+import org.example.repository.BuildingRepository;
+import org.example.util.TransformerDtoBuilding;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class BuildingService {
+
+    @Autowired
+    protected BuildingRepository buildingRepository;
+
+    @Autowired
+    protected TransformerDtoBuilding transformerDtoBuilding;
+
+    @Autowired
+    protected ApartmentRepository apartmentRepository;
+
+    public List<Apartment> showApartments(int id) {
```

Comment:
`show` is not appropriate word for method call in the service. `find` is much better